media-libs/cal3d/cal3d-0.120.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="Skeletal based character animation library"
+DESCRIPTION="Cal3D is a skeletal based 3D character animation library written \
+in C++ in a platform-/graphic API-independent way. Originally designed to \
+be used in a 3D client for Worldforge, it evolved into a stand-alone product \
+which can be used in many different kinds of projects. It supports \
+combining animations, actions and morph targets (facial expressions) through \
+a \"mixer\"."
+HOMEPAGE="https://mp3butcher.github.io/Cal3D/"
+COPYRIGHT="2001-2006 Mekensleep
+	Laurent Desmecht
+	Loic Dachary
+	Bruno Heidelberger"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/mp3butcher/Cal3D/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="fce2f57128bbefbcd4cee2d45337165e94614bc839c84e9c22b9bd988f10a440"
+SOURCE_FILENAME="cal3d-$portVersion.tar.gz"
+SOURCE_DIR="Cal3D-$portVersion"
```

Comment:
```diff
-SOURCE_DIR="Cal3D-$portVersion"
+SOURCE_DIR="Cal3D-$portVersion/cal3d"
```

We can use this to drop `cd` statements if the build system doesn't use any files outside of the said directory.

Reply:
Should the patchset be rewritten? haikuporter is smart enough to use the correct path though ...